Formula/tensorflow_c.rb

```diff
@@ -0,0 +1,46 @@
+class TensorflowC < Formula
+  desc "C interface for Google's OS library for Machine Intelligence"
+  homepage "https://www.tensorflow.org/"
+  url "https://github.com/tensorflow/tensorflow.git", :tag => "v1.0.0",
+    :revision => "07bb8ea2379bd459832b23951fb20ec47f3fdbd4"
+
+  depends_on "bazel" => :build
+  depends_on "pkg-config" => :run
+
+  def install
+    ENV["PYTHON_BIN_PATH"]=`which python`.strip
+    ENV["CC_OPT_FLAGS"]="-march=native"
+    ENV["TF_NEED_JEMALLOC"]="1"
+    ENV["TF_NEED_GCP"]="0"
+    ENV["TF_NEED_HDFS"]="0"
+    ENV["TF_ENABLE_XLA"]="0"
+    ENV["USE_DEFAULT_PYTHON_LIB_PATH"]="1"
+    ENV["TF_NEED_OPENCL"]="0"
+    ENV["TF_NEED_CUDA"]="0"
+    system "./configure"
+
+    system "bazel", "build", "--compilation_mode=opt", "--copt=-march=native", "tensorflow:libtensorflow_c.so"
```

Comment:
@kali: Yes, these two libraries are the same thing and the fact that there are two blaze targets that do exactly the same thing was an accident. So the idea is to suggest usage of `llibtensorflow.so` and remove `libtensorflow_c.so`.

I recently updated the Rust bindings to use `libtensorflow.so` instead of `libtensorflow_c.so` in https://github.com/tensorflow/rust/pull/56 and https://github.com/tensorflow/rust/pull/60,  and then plan to remove the `libtensorflow_c.so` target completely.